AppiumForMac/Server/Controller/AfMSessionController.m

```diff
@@ -161,112 +164,112 @@ - (BOOL)finishSetup
 - (NSTimeInterval)loopDelay
 {
     NSNumber *value = [self getCookieWithName:kCookieLoopDelay][@"value"];
-//    NSLog(@"loopDelay: %f", [value floatValue]);
+    DDLogCInfo(@"loopDelay: %f", [value floatValue]);
     return [value floatValue];
 }
 
 - (NSTimeInterval)commandDelay
 {
     NSNumber *value = [self getCookieWithName:kCookieCommandDelay][@"value"];
-//    NSLog(@"commandDelay: %f", [value floatValue]);
+    DDLogCInfo(@"commandDelay: %f", [value floatValue]);
     return [value floatValue];
 }
 
 - (NSTimeInterval)implicitTimeout
 {
     NSNumber *value = [self getCookieWithName:kCookieImplicitTimeout][@"value"];
-//    NSLog(@"implicitTimeout: %f", [value floatValue]);
+    DDLogCInfo(@"implicitTimeout: %f", [value floatValue]);
     return [value floatValue];
 }
 
 - (NSTimeInterval)pageLoadTimeout
 {
     NSNumber *value = [self getCookieWithName:kCookiePageLoadTimeout][@"value"];
-//    NSLog(@"pageLoadTimeout: %f", [value floatValue]);
+    DDLogCInfo(@"pageLoadTimeout: %f", [value floatValue]);
     return [value floatValue];
 }
 
 - (NSTimeInterval)scriptTimeout
 {
     NSNumber *value = [self getCookieWithName:kCookieScriptTimeout][@"value"];
-//    NSLog(@"scriptTimeout: %f", [value floatValue]);
+    DDLogCInfo(@"scriptTimeout: %f", [value floatValue]);
     return [value floatValue];
 }
 
 - (float)mouseMoveSpeed
 {
     NSNumber *value = [self getCookieWithName:kCookieMouseSpeed][@"value"];
-//    NSLog(@"mouseMoveSpeed: %f", [value floatValue]);
```

Comment:
What's the point in commenting that?

Reply:
NSLOG is commented by default. I just used file logger to write into external logs.